docs/configuration.md

```diff
@@ -635,6 +635,20 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.ui.reverseProxy</code></td>
+  <td>false</td>
+  <td>
+    Enable running Spark Master as reverse proxy for worker and application UI. In this mode, Spark master will reverse proxy the worker and application UIs to enable access without requiring direct access to their UIs. Use it with caution, as worker and application UI will not be accessible directly, you will only be able to access it through spark master/proxy public URL. This setting affects all the workers and application UIs running in the cluster and must be set on all the workers, drivers and master.
```

Comment:
everywhere: "UIs" (since you're saying "worker an application")

"...without requiring direct access to their hosts."

"...you will only be able to access them..."

"...workers, drivers and masters." (Yes there can be more than one.)